### PR TITLE
Fix tab disabled issue

### PR DESCRIPTION
**Pull request by zoncrd**
Fix an issue occurring in the Tab component when disabled and styled with custom CSS classes.
The `tab--disabled` CSS class was appended to the component className without an initial space and was concatenated to the last custom CSS class if provided, consequently breaking the styling.

I had also a simple test to check if the disabled Tab component is findable by the `tab--disabled` CSS class.

It's my first PR, let me know if it's okay or if I have to do something more.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Ability to add tooltips to `Slider` and `RangeSlider`, which can be visible always or on hover. Tooltips also take a position argument. [#564](https://github.com/plotly/dash-core-components/pull/564)
 
 ### Fixed
-- Fixed `min_date_allowed` and `max_date_allowed` bug in `DatePickerRange` [#551]https://github.com/plotly/dash-core-components/issues/551)
+- Fixed `min_date_allowed` and `max_date_allowed` bug in `DatePickerRange` [#551](https://github.com/plotly/dash-core-components/issues/551)
 - Fixed unwanted `resize()` calls on unmounted `Graph`s [#534](https://github.com/plotly/dash-core-components/issues/534)
+- Fixed `tab--disabled` CSS class issue in `Tab` component with custom styling
```
Comment (alexcjohnson) [inline CHANGELOG.md]:
```suggestion
- Fixed `tab--disabled` CSS class issue in `Tab` component with custom styling [#568](https://github.com/plotly/dash-core-components/pull/568)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,8 +7,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Ability to add tooltips to `Slider` and `RangeSlider`, which can be visible always or on hover. Tooltips also take a position argument. [#564](https://github.com/plotly/dash-core-components/pull/564)
 
 ### Fixed
-- Fixed `min_date_allowed` and `max_date_allowed` bug in `DatePickerRange` [#551]https://github.com/plotly/dash-core-components/issues/551)
+- Fixed `min_date_allowed` and `max_date_allowed` bug in `DatePickerRange` [#551](https://github.com/plotly/dash-core-components/issues/551)
```
Comment (alexcjohnson) [inline CHANGELOG.md]: hah cc @wbrgss I thought you already fixed this, bad rebase in #564?

Comment (wbrgss) [reply CHANGELOG.md]: Yeah, I thought I fixed it, must be.

**Comment by alexcjohnson** [comment]
Hmm, not sure why percy isn't picking up this job. Would it need write access to the source repo? Merging anyway.

**Pull request merged**